src/app/clusters/level-control/level-control.cpp

```diff
@@ -520,6 +520,11 @@ static EmberAfStatus moveToLevelHandler(EndpointId endpoint, CommandId commandId
         return EMBER_ZCL_STATUS_FAILURE;
     }
 
+    if (level > EMBER_AF_PLUGIN_LEVEL_CONTROL_MAXIMUM_LEVEL)
+    {
+        return EMBER_ZCL_STATUS_MALFORMED_COMMAND;
```

Comment:
For commands, I think it should be EMBER_ZCL_STATUS_INVALID_COMMAND. `EMBER_ZCL_STATUS_MALFORMED_COMMAND` is INVALID_ACTION in Matter. When data is out of bounds for a field, it would be INVALID_COMMAND I think.

Right now, MALFORMED_COMMAND is used only in LevelControl and Scenes server for commands, and otherwise is used in IM handler for attributes. INVALID_COMMAND/InvalidCommand is used much more places.

Strongly suggest moving this to INVALID_COMMAND and updating usage, since this error is not Matter compliant in the context AFAIK

Reply:
Sure I can change it to EMBER_ZCL_STATUS_INVALID_COMMAND.